notifications_utils/jinja_templates/email_template.jinja2

```diff
@@ -52,7 +52,7 @@
                   <tr>
                     <td style="padding-left: 10px">
                       <img
-                        src="https://static.notifications.service.gov.uk/images/gov.uk_logotype_crown.png"
+                        src="https://upload.wikimedia.org/wikipedia/en/thumb/a/a4/Flag_of_the_United_States.svg/320px-Flag_of_the_United_States.svg.png"
```

Comment:
We should be kind of wikipedia and deploy a cdn for our static assets before we hammer them